Formula/cli53.rb

```diff
@@ -0,0 +1,22 @@
+class Cli53 < Formula
+  desc "command-line tool for Amazon Route 53"
+  homepage "https://github.com/barnybug/cli53"
+  url "https://github.com/barnybug/cli53/archive/0.7.4.tar.gz"
+  sha256 "3cb89e6aa91676ffd0577798a4b06b056667d18ad836de4fa31c0564ee48474e"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/barnybug"
+    ln_s buildpath, buildpath/"src/github.com/barnybug/cli53"
+
+    system "make", "build"
+
+    bin.install "cli53"
+  end
+
+  test do
+    system "#{bin}/cli53", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!